tests/Action/ExceptionActionTest.php

```diff
@@ -59,7 +59,7 @@ public function testActionWithUncatchableException()
         $serializer = $this->prophesize(SerializerInterface::class);
         $serializer->serialize($flattenException, 'jsonproblem')->willReturn();
 
-        $exceptionAction = new ExceptionAction($serializer->reveal(), ['jsonproblem' => ['application/problem+json'], 'jsonld' => ['application/ld+json']]);
+        $exceptionAction = new ExceptionAction($serializer->reveal(), ['jsonproblem' => ['application/problem+json'], 'jsonld' => ['application/ld+json']], [], true);
```

Comment:
I would let it is it was previously (to test that there is no BC breaks and that the previous signature is still accepted).